pkg/apis/core/types.go

```diff
@@ -2189,7 +2189,7 @@ type SecretKeySelector struct {
 
 // EnvFromSource represents the source of a set of ConfigMaps
```

Comment:
same here

Reply:
Thanks for spotting this, I've updated it to mention Secrets too, let me know if you want it updating further